yandex/structures.go

```diff
@@ -1042,21 +1042,20 @@ func flattenStaticRoutes(routeTable *vpc.RouteTable) *schema.Set {
 	return staticRoutes
 }
 
-func expandStaticRoutes(d *schema.ResourceData) ([]*vpc.StaticRoute, error) {
+func expandStaticRoutes(v interface{}) ([]*vpc.StaticRoute, error) {
 	staticRoutes := []*vpc.StaticRoute{}
 
-	if v, ok := d.GetOk("static_route"); ok {
-		routeList := v.(*schema.Set).List()
-		for _, v := range routeList {
-			sr, err := routeDescriptionToStaticRoute(v)
-			if err != nil {
-				return nil, fmt.Errorf("fail convert static route: %s", err)
-			}
-			staticRoutes = append(staticRoutes, sr)
+	if v == nil {
```

Comment:
It's not (easily) possible to verify the set's emptiness, so we check for nil. It won't catch the following set with an empty element.
```
static_route {}
```

Reply:
I've found a proper way to do this.
https://github.com/yandex-cloud/terraform-provider-yandex/pull/105/commits/850383b92ed92e15cbb15770eab85de4bc3e5775